elements/centos-kmods-sig/install.d/95-kmods-sig-packages

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash -lv
+
+if [[ ${DISTRO_NAME} == centos && -n ${DIB_CENTOS_KMODS_PKGLIST} ]]; then
+    dnf install -y "${DIB_CENTOS_KMODS_PKGLIST}"
```

Comment:
I don't think this should be quoted.  With quotes, is the parameter parsed as a single token?  It will be handled as one package name (with spaces in)

Reply:
Absolutely, thanks for catching that!